aoc/src/main/scala/aoc2015/Day03.scala

```diff
@@ -0,0 +1,60 @@
+package aoc2015
+
+import scala.io.Source
+import scala.collection.mutable.ListBuffer
+
+object Day03 extends App:
+  val lines: String =
+    Source
+      .fromFile("aoc/src/resources/aoc2015/Day03")
+      .getLines
+      .mkString
+
+  def generateDirections(): Int =
+    var evenDirections: String = ""
```

Comment:
usage of `var`s is frowned upon - see whether you can rewrite the `foreach` loop with a `foldLeft` that calculates the direction strings as immutable values while traversing the lines with indexes